### PR TITLE
Fix bug #388

### LINKED ISSUE #388: As a watcher of anime through using cli I would like to pass a --volume={1-150} to start mpv at that volume level when anime is loaded

### PR DESCRIPTION
**Pull request by 71zenith**
Added -e flag to pass arguments to the player. Works with mpv. Updated the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +22,14 @@ A cli to browse and watch anime. This tool scrapes the site [gogoanime](https://
 ## Usage
 
   ```text
-    ani-cli [-v | -i] [-q <quality>] [-d | -p <download_dir>] [<query>]
-    ani-cli [-v | -i] [-q <quality>] -c
+    ani-cli [-v | -i] [-q <quality>] -e [<arguments>] [-d | -p <download_dir>] [<query>]
+    ani-cli [-v | -i] [-q <quality>] -e [<arguments>] -c
     ani-cli -h | -D | -U | -V
+    ani-cli -e [<arguments>]
```
Comment (ura43) [inline README.md:28]: remove that line

File: README.md [hunk 2]
```diff
@@ -23,12 +22,14 @@ A cli to browse and watch anime. This tool scrapes the site [gogoanime](https://
 ## Usage
 
   ```text
-    ani-cli [-v | -i] [-q <quality>] [-d | -p <download_dir>] [<query>]
-    ani-cli [-v | -i] [-q <quality>] -c
+    ani-cli [-v | -i] [-q <quality>] -e [<arguments>] [-d | -p <download_dir>] [<query>]
```
Comment (ura43) [inline README.md:25]: Does is pass arguments to aria2c in this case?

**Review by port19x — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (71zenith) [reply README.md]: Oops. I meant to delete it but then only did it for the script.

File: README.md (hunk 2 above)
Comment (71zenith) [reply README.md:25]: It doesn't. Should I add it?

**Comment by ura43** [comment]
Also please bump the version to 1.2.3

File: README.md (hunk 2 above)
Comment (ura43) [reply README.md:25]: No, not needed but in that case don't add it to that line

**Comment by ura43** [comment]
Synopsis syntax is a bit weird. Mind if I lend you a hand?

**Comment by 71zenith** [comment]
> 

Definitely

**Comment by ura43** [comment]
Actually, could you add the option passing to aria2c?
I think I'd use that

**Comment by ura43** [comment]
Have to reapply my commits to the synopsis now but thanks

**Comment by ura43** [comment]
I think we're ready for merge now

**Comment by 71zenith** [comment]
guess so

**Comment by IceAsteroid** [comment]
Hi, good day. Could someone please tell what the current state of the feature is now? It seems the `-e` option represents a different usage.

```
      -e, --episode, -r, --range
        Specify the number of episodes to watch
```

**Comment by port19x** [comment]
That sounds very implausible: https://github.com/pystardust/ani-cli/blob/82574d2480ec83ff014c8924349f7e944ce7c992/ani-cli#L482-L486

**Pull request merged**